padinfo/core/button_info.py

```diff
@@ -69,10 +90,30 @@ def _get_sub_attr_multiplier(self, monster_model):
         return 1 / 3
 
     def to_string(self, monster, info):
-        return INFO_STRING.format(monster.monster_id, monster.name_en, info.main_damage, info.sub_damage,
-                                  info.total_damage,
-                                  info.main_damage_with_atk_latent, info.sub_damage_with_atk_latent,
-                                  info.total_damage_with_atk_latent)
+        damage_str = self._get_btn_damage(Fteam_buttons, card_buttons, info, monster)
+        return INFO_STRING.format(monster.monster_id, monster.name_en, info.main_damage, info.sub_damage, 
+                                  info.total_damage, 
+                                  info.main_damage_with_atk_latent, info.sub_damage_with_atk_latent, 
+                                  info.total_damage_with_atk_latent, damage_str)
+
+    def _get_btn_damage(self, tb, cb, info, monster):
+        # TODO: calculate with oncolor assist damage and ATK+ eq (Oversoul)
+        ret_str = ""
+        cb.sort(key=lambda x: x.mult)
+        tb.sort(key=lambda x: x.mult)
+        for x in cb:
```

Comment:
also more meaningful variable than `x` like use `card` or something - maybe not monster since this isn't an actual monster type